### ISSUE TITLE
Weird crash with NVDA that never triggered a crash dump or anything and popped this up.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#5849]
This seems similar to another bug I received with NVDA the other day, and I will open a separate issue if I can pinpoint that one down and get some recon from NVDA on that crash.
This occurred when I was trying to open python's directory.
I went to the start menu, and started to type c:\ and then pressed enter. When I pressed enter, NVDA became completely and absolutely unresponsive, and the magic alt+ctrl+n didn't reboot it. Then I tried alt+ctrl+delete and NVDA worked fine on the login screen. Then, I escaped and tried some other desperate measures such as forcefully killing NVDA from the task manager which failed. The last thing I did (because my computer was completely frozen at this point) was to go back to the alt+ctrl+del menu and select log out. Then, NVDA gave a message saying that the program has terminated due too an error, (the whole dialog was able to be coppied thanks to windows allowing dialogs to be copied with ctrl+c). I can't find a crash dump (I have one in my temp folder, but I don't know when it was generated and if it was a crash from some time ago or not).  It looks like another flaky UIA component, b but I amm actually not positive.

IO - speech._speakSpellingGen (07:20:26):
Speaking character u'p'
IO - inputCore.InputManager.executeGesture (07:20:26):
Input: kb(laptop):enter
IO - speech.speak (07:20:27):
Speaking [u'c:\\']
DEBUGWARNING - watchdog._watcher (07:20:27):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 191, in <module>
  File "core.pyc", line 382, in main
  File "wx\_core.pyc", line 8657, in MainLoop
  File "wx\_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 353, in Notify
  File "queueHandler.pyc", line 83, in pumpAll
  File "queueHandler.pyc", line 50, in flushQueue
  File "eventHandler.pyc", line 61, in _queueEventCallback
  File "eventHandler.pyc", line 143, in executeEvent
  File "eventHandler.pyc", line 91, in __init__
  File "eventHandler.pyc", line 98, in next
  File "NVDAObjects\__init__.pyc", line 829, in event_typedCharacter
  File "NVDAObjects\__init__.pyc", line 816, in _reportErrorInPreviousWord
  File "NVDAObjects\UIA\__init__.pyc", line 256, in getTextWithFields
  File "NVDAObjects\UIA\__init__.pyc", line 206, in _getTextWithFieldsForRange

Message box dump:

---------------------------
nvda.exe - Application Error
---------------------------
The exception unknown software exception (0xc0000409) occurred in the application at location 0x759e6c8f.

Click on OK to terminate the program
---------------------------
OK   
---------------------------

Now that I look at it, it was in the spell check thingy that was just added (which is odd).

**Comment by Brian1Gaff** [comment]
Hmm, I had a complete system reset in xp with word the other day in the spellchecker, it muttered something about not finding a dll file for dot then restarted.
 This was word 2003 and the latest next version. However, its not done it since so....
 Brian

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
  ----- Original Message ----- 
  From: derekriemer 
  To: nvaccess/nvda 
  Sent: Saturday, April 02, 2016 2:32 PM
  Subject: [nvaccess/nvda] Weird crash with NVDA that never triggered a crash dump or anything and popped this up. (#5849)


  This seems similar to another bug I received with NVDA the other day, and I will open a separate issue if I can pinpoint that one down and get some recon from NVDA on that crash.
  This occurred when I was trying to open python's directory.
  I went to the start menu, and started to type c:\ and then pressed enter. When I pressed enter, NVDA became completely and absolutely unresponsive, and the magic alt+ctrl+n didn't reboot it. Then I tried alt+ctrl+delete and NVDA worked fine on the login screen. Then, I escaped and tried some other desperate measures such as forcefully killing NVDA from the task manager which failed. The last thing I did (because my computer was completely frozen at this point) was to go back to the alt+ctrl+del menu and select log out. Then, NVDA gave a message saying that the program has terminated due too an error, (the whole dialog was able to be coppied thanks to windows allowing dialogs to be copied with ctrl+c). I can't find a crash dump (I have one in my temp folder, but I don't know when it was generated and if it was a crash from some time ago or not). It looks like another flaky UIA component, b but I amm actually not positive.

  IO - speech.speakSpellingGen (07:20:26):
  Speaking character u'p'
  IO - inputCore.InputManager.executeGesture (07:20:26):
  Input: kb(laptop):enter
  IO - speech.speak (07:20:27):
  Speaking [u'c:\']
  DEBUGWARNING - watchdog.watcher (07:20:27):
  Trying to recover from freeze, core stack:
  File "nvda.pyw", line 191, in 
  File "core.pyc", line 382, in main
  File "wx_core.pyc", line 8657, in MainLoop
  File "wx_core.pyc", line 7952, in MainLoop
  File "core.pyc", line 353, in Notify
  File "queueHandler.pyc", line 83, in pumpAll
  File "queueHandler.pyc", line 50, in flushQueue
  File "eventHandler.pyc", line 61, in _queueEventCallback
  File "eventHandler.pyc", line 143, in executeEvent
  File "eventHandler.pyc", line 91, in __init
  File "eventHandler.pyc", line 98, in next
  File "NVDAObjects__init__.pyc", line 829, in event_typedCharacter
  File "NVDAObjects__init__.pyc", line 816, in reportErrorInPreviousWord
  File "NVDAObjects\UIA__init_.pyc", line 256, in getTextWithFields
  File "NVDAObjects\UIA__init__.pyc", line 206, in _getTextWithFieldsForRange

  Message box dump:


------------------------------------------------------------------------------

  nvda.exe - Application Error
  The exception unknown software exception (0xc0000409) occurred in the application at location 0x759e6c8f.

  Click on OK to terminate the program
  OK 
  Now that I look at it, it was in the spell check thingy that was just added (which is odd).

**Comment by jcsteh** [comment]
Looks like NVDA crashed while trying to make a UIA text pattern query. Assuming you're running next, it looks like it crashed in IUIAutomationTextRange.Clone. Unfortunately, without a dump, there's nothing more we can glean from this, so I'm closing as can't fix unless we get a reliable way to reproduce this or a dump.

**Comment by derekriemer** [comment]
Did my devinfo get through?

On 4/5/2016 10:03 PM, James Teh wrote:
>
> Closed #5849 <https://github.com/nvaccess/nvda/issues/5849>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/5849#event-616164297>
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194
